homeassistant/components/light/hyperion.py

```diff
@@ -0,0 +1,131 @@
+"""
+homeassistant.components.light.hyperion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Hyperion remotes.
+
+Configuration:
+
+To connect to [a Hyperion server](https://github.com/tvdzwan/hyperion) you
+will need to add something like the following to your configuration.yaml file:
+
+light:
+    platform: hyperion
+    host: 192.168.1.98
+    port: 19444
+
+The JSON server port is 19444 by default.
+"""
+import logging
+import socket
+import json
+
+from homeassistant.const import CONF_HOST
+from homeassistant.components.light import (Light, ATTR_RGB_COLOR)
+
+_LOGGER = logging.getLogger(__name__)
+REQUIREMENTS = []
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """ Sets up a Hyperion server remote """
+    host = config.get(CONF_HOST, None)
+    port = config.get("port", 19444)
+    device = Hyperion(host, port)
+    if device.setup():
+        add_devices_callback([device])
```

Comment:
Let's also `return True` here. It is not being used right now but might be in the future.